archive/respack.ksy

```diff
@@ -0,0 +1,28 @@
+meta:
+  id: respack
+  title: ResPack
+  license: CC0-1.0
+  encoding: UTF-8
+  endian: le
+doc: |
+  Resources file found in CPB firmware archives, mostly used on older CoolPad
+  phones and/or tablets. The only observed files are called "ResPack.cfg".
+seq:
+  - id: header
+    type: header
+  - id: json
+    size: header.len_json
+    type: strz
```

Comment:
According to https://izden.wodemo.net/browse/390948-46a74bb046319c0216e646858def910b/assets/cfg/a/ResPack.cfg, this doesn't appear to be null-terminated:

```suggestion
  - id: json
    size: header.len_json
    type: str
```

Reply:
Yes, indeed, you are right. Because of `size` it doesn't seem to be a problem though.